tests/units/modules/test_kubevirt_vm_status.py

```diff
@@ -7,7 +7,7 @@
 from ansible.module_utils._text import to_bytes
 
 # FIXME: paths/imports should be fixed before submitting a PR to Ansible
```

Comment:
Does this change solve the FIXME ?

Reply:
Not yet, I am working on it in another patch.